Documentation/project-docs/how-to-track-changes.md

```diff
@@ -0,0 +1,164 @@
+wrote up what I learned. If you discover this doc is inaccurate or incomplete, 
+kindly update it! 
+
+## How Release works from 10,000 ft
+
+1. Every night an automated build runs on the source of each dotnet repo and
+produces some binaries that are placed in NuGet packages on MyGet. For example
+[this CoreCLR package](https://dotnet.myget.org/feed/dotnet-core/package/nuget/runtime.win-x64.Microsoft.NETCore.Runtime.CoreCLR)
+is one of the packages produced by the 
+[CoreCLR repo](https://github.com/dotnet/coreclr).
+These packages are generally not intended for distribution to customers, they 
+are only containers to exchange binaries.
+2. Every night a seperate automated build for the 
+[core-setup](https://github.com/dotnet/core-setup) repo produces 
+[installers](https://github.com/dotnet/core/blob/master/daily-builds.md)
+and the [Microsoft.NetCore.App NuGet package](https://dotnet.myget.org/feed/dotnet-core/package/nuget/Microsoft.NETCore.App).
+These packages aggregate the binaries that were built from many other repos 
+and this is what customers use - though more often Preview and RTM builds 
+instead of the less tested daily builds. 
+The daily builds of installers and Microsoft.NetCore.App do not necessarily
+have the latest builds from each constiuent repo, thus the build versions
+from individual repos may not match the build version of Microsoft.NetCore.App 
+or an installer that aggregated them.
+
+## Our starting point
+
+In my case I had a git commit: [435a69a9aca037f348a09e00dffef8a0888682a7 
+in the coreclr repo](https://github.com/dotnet/coreclr/commit/435a69a9aca037f348a09e00dffef8a0888682a7) 
+and I wanted to know where this code had a propogated to. Alternatively
+you may be starting from a particular package or installer and wondering
+what source it contains. Lets start exploring!
+
+
+## Where are the daily build NuGet packages published?
+
+For many of the dotnet repos such as CoreCLR the daily builds are posted to the 
+[dotnet-core](https://dotnet.myget.org/gallery/dotnet-core) feed. Other repos or 
+specific branches might publish elsewhere in which case hopefully you can find 
+that in the repo docs or by searching [the gallery](https://dotnet.myget.org/gallery) 
+for a appropriate sounding feed name.
+
+
+## What (intermediate) NuGet packages does a given repo produce?
+
+Whenever a dotnet repo publishes new builds it makes a commit in the 
+[dotnet/versions repo](https://github.com/dotnet/versions) indicating which 
+packages got published. For example during the night after I merged my 
+example CoreCLR change 
+[this commit](https://github.com/dotnet/versions/commit/cf8930fbe52e5eacf8ab0d7fb06f032d19cda5d5#diff-5f6099c37f777c410c4397b3f1e38870)
+showed up in dotnet/versions. Looking at the 
+[Last_Build_Packages.txt](https://github.com/dotnet/versions/blob/master/build-info/dotnet/coreclr/master/Last_Build_Packages.txt) 
+that commit edited you can see a list of NuGet packages that CoreCLR repo 
+publishes.
+
```

Comment:
Sentence is odd. Not sure what 'that commit edited' is about. Also, should it be explicitly mentioned that Last_Build_Packages.txt really is volatile or is the file name enough to clue the reader in?

Reply:
Hopefully the new sentence flows better and the 'was modified in that commit' gives a solid hint this file has volatile contents : )